daemon/openrazer_daemon/misc/ripple_effect.py

```diff
@@ -30,6 +30,8 @@ def __init__(self, parent, device_number):
         self._shutdown = False
         self._active = False
 
+        self._matrixdims = self._parent._parent.MATRIX_DIMS
+
         self._keyboard_grid = KeyboardColour()
```

Comment:
It looks like you forgot to add the row + column arguments here

Reply:
Yep, sorry